fossunited/foss_hackathon/doctype/foss_hackathon_localhost/foss_hackathon_localhost.py

```
@@ -40,6 +40,7 @@ class FOSSHackathonLocalHost(WebsiteGenerator):
         parent_hackathon: DF.Link
         route: DF.Data | None
         slug: DF.Data | None
+        sponsor_list: DF.Table[FOSSEventSponsor]
```

Ruff (0.14.14)

[error] Undefined name `FOSSEventSponsor`

(F821)